opennms-provision/opennms-provision-shell/src/main/java/org/opennms/netmgt/provision/requisition/command/ShowImport.java

```diff
@@ -87,7 +89,7 @@ public Object execute() throws Exception {
             try {
                 try {
                     Requisition requisition = future.get(1, TimeUnit.SECONDS);
-                    if (xmlOutput) {
+                    if (xmlOutput && !Strings.isNullOrEmpty(requisition.getForeignSource())) {
```

Comment:
Why should we avoid marshaling if the FS is null?

Reply:
I made this change a whie ago when working on pris, I  think it is throwing null pointer exception if there is no valid requisition.  It is hiding the actual errors. Let me check again

Reply:
doesn't go into marshal if there is no requisition. Might be because of something in PRIS that I was getting exception.  Removed